pkg/noobaaaccount/reconciler.go

```diff
@@ -373,6 +374,30 @@ func (r *Reconciler) CreateNooBaaAccount() error {
 		r.Logger.Errorf("got error on NooBaaAccount creation. error: %v", err)
 		return err
 	}
+	// create join secret conatining auth token for remote noobaa account
+	if r.NooBaaAccount.Spec.Role != "" && r.NooBaaAccount.Spec.Role == "noobaa-remote" {
+		res, err := r.NBClient.CreateAuthAPI(nb.CreateAuthParams{
+			System: r.Request.Name,
+			Role:   "remote-operator",
```

Comment:
Do we need to distinguish between `operator` and `remote-operator`? in any case, you need support for that in the core side

https://github.com/noobaa/noobaa-core/blob/a6e8837cb96be68fb76f8e5b007b28fe80045001/src/server/system_services/schemas/role_schema.js#L26-L30

Reply:
I'll go over noobaa-core, to see if now when we have access control by owner_id, maybe we don't need to distinguish between operator and remote-operator

Reply:
Using existing `operator` role